src/model/company.profile.ts

```diff
@@ -0,0 +1,19 @@
+export interface PTFCompanyProfile {
+    companyName: string;
+    companyNumber: string;
+    companyStatus: string;
+    companyType: string;
+    address: {
+        line_1: string;
+        line_2: string;
+        postCode: string;
+    };
+    accountsDue: string;
+    accountingPeriodStartOn: string;
+    accountingPeriodEndOn: string;
+    isAccountsOverdue: boolean;
+    isCSOverdue: boolean;
```

Comment:
Would suggest not shortening ConfirmationStatement to CS, but up to you.

Reply:
Godd idea I'll make this more descriptive